### ISSUE TITLE
about train loss

### ISSUE THREAD

**Issue by harrylin-hyl** [xingyizhou/CenterNet#743]
when I train model in coco/pascal， the loss is about 0.9, can not be smaller. is that normal?

**Comment by xingyizhou** [comment]
I didn't monitor the loss value. It should be OK if the mAP is expected.

**Comment by sisrfeng** [comment]
I think using the mAP instead of the val_loss as the critieria for saving model_best is more reasonable.

> I didn't monitor the loss value. It should be OK if the mAP is expected.

**Comment by sisrfeng** [comment]
How about using the mAP as the train_loss directly?

https://github.com/cccorn/AP-loss

用于目标检测的 AP loss：
AP 被当做评价指标，但由于其不可微和非凸性，难以用作优化函数，本文的方法有四个特点：
1）本文方法可以被用于任何可微线性或可微非线性模型，如神经网络，其他的那些方法，如[37,19] 仅仅可以在线性 SVM 模型中起作用。
2）本文方法直接优化 AP- loss，[20]会导致 loss 出现 gap
3）本文方法不是近似的梯度，且不受 [34,9] 中的目标函数非凸性的影响
4）本文方法可以端到端的训练检测器
